metadatastore/conf.py

```diff
@@ -3,18 +3,29 @@
 import logging
 
 logger = logging.getLogger(__name__)
-filename = os.path.join(os.path.expanduser('~'), '.config', 'metadatastore',
-                        'connection.yml')
-if os.path.isfile(filename):
-    with open(filename) as f:
-        connection_config = yaml.load(f)
-    logger.debug("Using db connection specified in config file. \n%r",
-                 connection_config)
-else:
-    connection_config = {
-        'database': 'test',
-        'host': "localhost",
-        'port': 27017,
-        }
-    logger.debug("Using default db connection. \n%r",
-                 connection_config)
+
+def load_configuration(name, prefix, fields):
+    filenames = [os.path.join('etc', name + '.yml'),
+                 os.path.join(os.path.expanduser('~'), '.config',
```

Comment:
These files should not be automatically be created but provide you a manual way to over-ride the packaged configuration.  Ex our dev accounts have a config which always points to a dev server so we have less chance of accidentally blitting a production server.